src/scss/layout/_team-section.scss

```diff
@@ -0,0 +1,87 @@
+.team {
+    background-color: var(--white);
+
+
+    &__heading {
+        @include fontStyle(var(--waikawa-gray), 24px);
+        line-height: 28px;
+        padding: 5.6rem 5.2rem 2.3rem 5.2rem;
+    }
+
+    &__container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 0 5.2rem 2.3rem 5.2rem;
+        margin-left: 0;
+        gap: 2.7rem 2rem;
+
+        @include breakpoint('md') {
+            display: grid;
+            grid-template-columns: repeat(2, 1fr);
+            justify-items: center;
+            grid-gap: 2.7rem 2rem;
+        }
+
+        @include breakpoint('lg') {
+            grid-template-columns: repeat(3, 1fr);
+        }
+
+        @include breakpoint('xl') {
+            grid-template-columns: repeat(4, 1fr);
+        }
+    }
+
+    &__card {
+        
+        background: var(--white);
+        box-shadow: 0 4px 4px var(--gray-shadow);
+        border-radius: 16px;
+        max-width: 360px;
+        min-height: 440px;
+        
+        &--centered-content {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+    }
+
+    &__image {
+        width: 100%;
+        height: 230px;
+        border-radius: 16px 16px 0 0;
```

Comment:
you can remove this `border-radius` using `overflow: hidden` in the `.team__card container`.